doc/source/cheat_sheet/cheat_sheet.qmd

```diff
@@ -0,0 +1,275 @@
+---
+title: PyMAPDL cheat sheet
+format: cheat_sheet-pdf
+version: main
+footer: PyMAPDL
+footerlinks:
+  - urls: 'https://mapdl.docs.pyansys.com/version/stable/'
+    text: Documentation
+  - urls: 'https://mapdl.docs.pyansys.com/version/stable/getting_started/index.html'
+    text: Getting started
+  - urls: 'https://mapdl.docs.pyansys.com/version/stable/examples.html'
+    text: Examples
+  - urls: 'https://mapdl.docs.pyansys.com/version/stable/api/index.html'
+    text: API reference
+  - urls: 'https://github.com/ansys/pymapdl/discussions'
+    text: Discussions
+  - urls: 'https://github.com/ansys/pmapdl/issues'
+    text: 'Issues'
+execute:
+    output: false
+    eval: true
+
+latex-clean: true
+latex-auto-install: true
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .qmd
+      format_name: quarto
+      format_version: '1.0'
+      jupytext_version: 1.16.1
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+# Connect to MAPDL remote session
+
+Launch and connect to MAPDL. You can specify the version you want to use with
+``version`` argument.
+```{python}
+from ansys.mapdl.core import launch_mapdl
+mapdl = launch_mapdl(version=242)
+```
+
+Exit a session
+```{python}
+# Exit it
+mapdl.exit()
+```
+
+Connect to an existing instance at IP address ``192.168.1.30`` and port ``50001``:
+```{python}
+#| eval: false
+mapdl = launch_mapdl(start_instance=False, ip='192.168.1.30', port=50001)
+```
+
+Create and exit a pool of instances:
+```{python}
+#| eval: false
+# Create a pool of 10 instances
+from ansys.mapdl.core import pool
+mapdl_pool = pool.MapdlPool(5)
+# Exit the pool
+mapdl_pool.exit()
+```
+
+# PyMAPDL CLI
+
+Access MAPDL instances through CLI
+```{python}
+#| eval: false
+# Start an MAPDL instance at port ``50051``
+pymapdl start --port 50051
+# List the current MAPDL instances and processes
+pymapdl list
+# Stop all the MAPDL instances
+pymapdl stop --all
+```
+
+# PyMAPDL commands
+
+PyMAPDL commands are Python statements that act as a
+wrapper for APDL commands. For example: ``ESEL,s,type,1`` 
+is translated as
+```{python}
+#| echo: false
+mapdl = launch_mapdl()
+```
+```{python}
+mapdl.esel('s', 'type', vmin=1)
+```
+
+Help for the MAPDL class functions is accessible with:
+```{python}
+help(mapdl.esel)
+```
+
+```{python}
+#| echo: false
+mapdl.exit()
+```
+
+Most of the time, commands that start with \* or \/ have these characters
+removed:
+```{python}
+#| eval: false
+mapdl.prep7() # /PREP7
+mapdl.get() # *GET
+```
+
+In cases where removing * or / causes conflicts with other
+commands, a prefix ``slash`` or ``star`` is added:
+```{python}
+#| eval: false
+mapdl.solu() # SOLU
+mapdl.slashsolu() # /SOLU
+
+mapdl.vget() # VGET
+mapdl.starvget() # *VGET
```

Comment:
We are trying to discourage the usage of `starvget`, to use `get_array` instead. Same with `startget` and `get_value`. Actually, it will be nice to have them in this cheat sheet too.

```suggestion

```

Reply:
Great idea!